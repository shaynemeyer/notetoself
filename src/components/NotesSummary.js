import React from 'react';
import { Link } from 'react-router-dom';

const NotesSummary = () => {
  return (
    <div className="page-header">
      <div className="content-container">
        <h1>My Notes</h1>
        <div className="page-header__actions">
          <Link className="button" to="/create">Add Note</Link>
        </div>
      </div>
    </div>
  );
};

export default NotesSummary;
