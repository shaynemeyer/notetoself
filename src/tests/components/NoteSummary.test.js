import React from 'react';
import { shallow } from 'enzyme';
import NotesSummary from '../../components/NotesSummary';

test('should render NotesSummary', () => {
  const wrapper = shallow(<NotesSummary />);
  expect(wrapper).toMatchSnapshot();
});
