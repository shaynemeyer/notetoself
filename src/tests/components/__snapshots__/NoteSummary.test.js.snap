// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotesSummary 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1>
      My Notes
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        replace={false}
        to="/create"
      >
        Add Note
      </Link>
    </div>
  </div>
</div>
`;
